lib/credentials_manager/appfile_config.rb

```diff
@@ -115,6 +115,10 @@ def for_platform(platform_name, &block)
 
     # Private helpers
 
+    def platform_specified?
+      ENV["FASTLANE_PLATFORM_NAME"] && !ENV["FASTLANE_PLATFORM_NAME"].strip.empty?
```

Comment:
i think `unless platform_specified? ` reads better and would make the comment on the following line unnecessary, but made change